lib/config/configuration.go

```diff
@@ -156,10 +156,6 @@ func ApplyFileConfig(fc *FileConfig, cfg *service.Config) error {
 		cfg.AuthServers = make([]utils.NetAddr, 0, len(fc.AuthServers))
 		for _, as := range fc.AuthServers {
 			addr, err := utils.ParseHostPortAddr(as, defaults.AuthListenPort)
-			if err != nil {
```

Comment:
I would not remove the error handling here, in case if no auth servers are provided in file config, the list should be empty.

Reply:
Yes, sorry. Its not exactly related to this PR, but while trying to understand how it all works I noticed that there is an error check just below with proper error message that will never be executed. This one seems extraneous. Reverted my change.